geetools/ee_profiler.py

```diff
@@ -0,0 +1,94 @@
+"""A profiler context manager for Earth Engine Python API."""
+from __future__ import annotations
+
+import io
+import re
+
+import ee
+
+from .accessors import _register_extention
+
+
+@_register_extention(ee)
```

Comment:
I'm strongly advocating so the GEE package self detect and install the extention packages like pytest is doing for plugins. If this becomes a reality at some point you would not need to import "geetools" by yourselve anymore as it will just be part of the environment. so instead of this I can make it into an `ee.geetools.Profiler` ?

Reply:
Actually `ee.geetools.Profiler` was my first choice, but I though it was going to be a bit more complicated, and since user needs to import `geetools`, I though on the second (simpler) option. But wow, that's great news! kudos 🙌

Reply:
solved https://github.com/gee-community/geetools/pull/358/commits/e535f0c0a39abd715c97c40af7b6f00517e56959